data/templates/ignition.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ignition": { "version": "2.0.0" },
+  "storage": {
+    "files": [{
+      "filesystem": "root",
+      "path": "/root/ignition-RackHD.txt",
+      "mode":400,
+      "contents": { "source": "data:text/plain;charset=utf-8;base64,WW91IG1heSBjdXN0b21pemUgeW91ciBDb3JlT1MgaW5zdGFsbGF0aW9uIHVzaW5nIElnbml0aW9uIGJ5IGNvbmZpZ3VyaW5nIHRoZSBpZ25pdGlvblNjcmlwdFVyaSB2YXJpYWJsZSB0byBwb2ludCB0byBhIGN1c3RvbSBJZ25pdGlvbiBmaWxlIHByaW9yIHRvIGV4ZWN1dGluZyB0aGUgUmFja0hEIFRhc2suT3MuSW5zdGFsbC5Db3JlT1MgdGFzay4=" }
```

Comment:
Hello @yyscamper, that string is base64 encoded (per guidance in the Ignition documentation). When decoded (feel free to verify if you're concerned), it says the following:

> You may customize your CoreOS installation using Ignition by configuring the ignitionScriptUri variable to point to a custom Ignition file prior to executing the RackHD Task.Os.Install.CoreOS task.

Please let me know if I can help clarify anything else.